lib/prop/options.rb

```diff
@@ -17,6 +17,15 @@ def self.build(options)
       raise RuntimeError.new("Invalid threshold setting") unless result[:threshold] > 0
       raise RuntimeError.new("Invalid interval setting")  unless result[:interval] > 0
 
+      if result[:leaky_bucket]
+        result[:burst_rate] = result[:burst_rate].to_i
+        result[:strategy]   = Prop::LeakyBucketStrategy
+
+        raise RuntimeError.new("Invalid burst rate setting") unless result[:burst_rate] > result[:threshold]
```

Comment:
`unless >` -> `if <`
maybe also be more specific like "burst_rate is higher then threshold"